### DISCUSSION TITLE
Tooltip Color

### DISCUSSION THREAD

**Discussion by TidharHayute** [radix-ui/themes#137] [Q&A]
Hey there! I wanted to ask if it's possible to change the tooltip color?
I tried adding { style: "#151515" } but it only changed the background color of the text, and the little arrow below had the same default color. 

Thanks!

**Comment by vladmoroz** [comment]
Hey, take a look at how the tooltip styles are implemented

https://github.com/radix-ui/themes/blob/main/packages/radix-ui-themes/src/components/tooltip.css

You could override them via matching classes, or override the CSS variable used for the background there.

**Reply by TidharHayute** [reply]
Thank you! I will try this right now
